src/main/java/com/google/sps/Course.java

```diff
@@ -1,14 +1,20 @@
+package com.google.sps;
```

Comment:
This seems like a data model for the Course resource. I'd suggest creating a new package/folder named 'model'. Your file would look like: src/main/java/com/google/sps/model/Course.java